core/src/main/java/org/axonframework/queryhandling/QueryBus.java

```diff
@@ -104,17 +89,68 @@
 
     /**
      * Dispatch the given {@code query} to a single QueryHandler subscribed to the given {@code query}'s
-     * queryName/initialResponseType/updateResponseType.
+     * queryName/initialResponseType/updateResponseType. The result is lazily created and until there is a subscription
```

Comment:
This second sentence seems a little off.

Shouldn't it be something like `The result is lazily created and there will be no execution of the query handler before there is a subscription to the initial result.`?